option/internaloption/internaloption.go

```diff
@@ -126,6 +126,19 @@ func (w withDefaultScopes) Apply(o *internal.DialSettings) {
 	copy(o.DefaultScopes, w)
 }
 
+// WithDefaultUniverseDomain returns a ClientOption that sets the default universe domain.
+//
+// It should only be used internally by generated clients.
```

Comment:
Can you add a doc line below this one:

```
//
// 
// This is similar to WithUniverseDomain, but allows us to determine whether the user has overridden the default universe domain.

Reply:
I saw this comment on `WithDefaultEndpoint`, it seems useful.

Reply:
Done.